lib/plugins/features/movies/controller.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Movie = require('../../../models/movie');
+
+exports.create = (payload) => {
+  return new Movie().save(payload)
+  .then((movie) => new Movie({ id: movie.id }).fetch());
```

Comment:
you should be able to just return `movie` here

Reply:
jessica mentioned that Movie().save() returns a promise and that it was just the syntax of bookshelf that did it this way. do you know more about this?

Reply:
oh wait, i think im wrong... i thought we had to save the movie and that does not return a model so we have to fetch it again. but actually i think you can do you what lucas said

Reply:
`.save()` returns the Bookshelf object once the promise completes, so you wouldn't even need to refetch the movie if you wanted to do anything with it. If you return it in this response the promise completes; so if you make that change and run the tests again they should still pass